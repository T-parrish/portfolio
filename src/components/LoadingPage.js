import React from 'react';

const LoadingPage = () => (
    <div className="loader-container">
        <div className="animation-wrapper">
            <div className="wrapper">
                <div className="loaderBall__one"></div>
                <div className="shadow__one"></div>
            </div>
            <div className="wrapper">
                <div className="loaderBall__two"></div>
                <div className="shadow__two"></div>
            </div>
            <div className="wrapper">
                <div className="loaderBall__three"></div>
                <div className="shadow__three"></div>
            </div>
            <div className="wrapper">
                <div className="loaderBall__four"></div>
                <div className="shadow__four"></div>
            </div>    
            <div className="wrapper">
                <div className="loaderBall__five"></div>
                <div className="shadow__five"></div>
            </div>
        </div>
        <div className="loader__title">
            <h1>Loading...</h1>
        </div>
    </div>
);

export default LoadingPage;