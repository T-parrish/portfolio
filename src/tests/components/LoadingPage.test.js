import React from 'react';
import { shallow } from 'enzyme';
import LoadingPage from '../../components/LoadingPage';

test('should correctly render Login Page component', () => {
    const wrapper = shallow(<LoadingPage />);
    expect(wrapper).toMatchSnapshot();
});