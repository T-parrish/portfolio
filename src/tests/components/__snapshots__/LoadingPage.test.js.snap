// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Login Page component 1`] = `
<div
  className="loader-container"
>
  <div
    className="animation-wrapper"
  >
    <div
      className="wrapper"
    >
      <div
        className="loaderBall__one"
      />
      <div
        className="shadow__one"
      />
    </div>
    <div
      className="wrapper"
    >
      <div
        className="loaderBall__two"
      />
      <div
        className="shadow__two"
      />
    </div>
    <div
      className="wrapper"
    >
      <div
        className="loaderBall__three"
      />
      <div
        className="shadow__three"
      />
    </div>
    <div
      className="wrapper"
    >
      <div
        className="loaderBall__four"
      />
      <div
        className="shadow__four"
      />
    </div>
    <div
      className="wrapper"
    >
      <div
        className="loaderBall__five"
      />
      <div
        className="shadow__five"
      />
    </div>
  </div>
  <div
    className="loader__title"
  >
    <h1>
      Loading...
    </h1>
  </div>
</div>
`;
